modules/avro-exporter/src/main/scala/io/janstenpickle/trace4cats/avro/AvroSpanExporter.scala

```diff
@@ -112,7 +117,11 @@ object AvroSpanExporter {
     blocker: Blocker,
     host: String = agentHostname,
     port: Int = agentPort,
+    numFibers: Int = 1
```

Comment:
Could you modify the tests to add a case where multiple fibers are used please?

Reply:
Yes, a bit later. How important is the order of spans being sent by Exporter? Multiple fibers will be sending batches out of the queue order.

Reply:
@janstenpickle done
I also noticed that the existing tests were using `Batch[List]` which arb instance returned zero-sized lists (there's minSuccessful = 1 configuration). I changed it to `Batch[Chunk]` as it's used in other tests